src/lib/bootloader/grub2_widgets.rb

```diff
@@ -113,30 +114,54 @@ def store
   end
 
   # Represents decision if smt is enabled
-  class Smt < CWM::CheckBox
+  class CpuMitigationsWidget < CWM::ComboBox
     include Grub2Widget
 
     def initialize
       textdomain "bootloader"
     end
 
     def label
-      _("Disable Simultaneous &Multithreading")
+      _("CPU Mitigations")
+    end
+
+    def items
+      ::Bootloader::CpuMitigations::ALL.map do |m|
+        [m.value, m.to_human_string]
+      end
     end
 
     def help
+      # TODO: adapt
       _(
-        "<p><b>Disable Simultaneous Multithreading</b><br>\n" \
-          "To disable sharing physical cores by more virtual ones."
+        "<p><b>CPU Mitigations</b><br>\n" \
+          "The option selects which default settings should be used for CPU " \
```

Comment:
`\n` are missing at the end of **every** line. 
```suggestion
          "The option selects which default settings should be used for CPU\n" \
```

Reply:
it is rich text so it does not matter.

Reply:
It does matter to the translators who will have to read it with their eyes. Likewise we do format HTML code even if the browser renders it anyway